src/events/actions/updateWebsiteActions/generateVoxxrinJson.ts

```diff
@@ -0,0 +1,210 @@
+                secondaryHex: event.colorSecondary || '#3880FF',
+                secondaryContrastHex: hexContrastColor(event.colorSecondary || '#3880FF'),
+                tertiaryHex: event.colorBackground || '#202020',
+                tertiaryContrastHex: hexContrastColor(event.colorBackground || '#202020'),
+            },
+        },
+        supportedTalkLanguages: supportedTalkLanguages,
+        rooms: event.tracks.map((track) => ({ id: track.id, title: track.name })),
+        talkTracks: event.categories.map((category) => ({
+            id: category.id,
+            title: category.name,
+            themeColor: category.color || '#000000',
+        })),
+        talkFormats: event.formats.map((format) => ({
+            id: format.id,
+            title: format.name,
+            duration: `PT${format.durationMinutes}m`,
+            themeColor: '#000000',
+        })),
+        features: {
+            favoritesEnabled: true,
+            roomsDisplayed: true,
+            remindMeOnceVideosAreAvailableEnabled: false,
+            showInfosTab: true,
+            showRoomCapacityIndicator: false,
+            hideLanguages: [],
+            ratings: {
+                scale: {
+                    enabled: false,
+                },
+                bingo: {
+                    enabled: false,
+                },
+                'free-text': {
+                    enabled: false,
+                },
+            },
+        },
+        sponsors: sponsors.map((sponsorCategory) => ({
+            type: sponsorCategory.name,
+            typeFontColor: event.color || '#000000',
+            typeBackgroundColor: event.colorBackground || '#E5E4E2',
+            sponsorships: sponsorCategory.sponsors
+                .map((sponsor) => ({
+                    name: sponsor.name,
+                    logoUrl: sponsor.logoUrl,
+                    href: sponsor.website || '',
+                }))
+                .filter((sponsor) => !!sponsor.href && sponsor.href.length > 0),
+        })),
+        sessions: {},
+        speakers: {},
+    }
+
+    sessions.forEach((session) => {
+        if (!session.showInFeedback) {
+            return
+        }
+
+        const tags = []
+
+        const categoryName = session.category
+            ? event.categories.find((category) => category.id === session.category)?.name
+            : null
+
+        if (categoryName) {
+            tags.push(categoryName)
+        }
+
+        voxxrinJson.sessions[session.id] = {
+            speakers: session.speakers,
+            tags: tags,
+            title: session.title,
+            id: session.id,
+            categoryName: categoryName,
+            categoryId: session.category,
+            formatId: session.format,
+            abstract: session.abstract,
+            trackId: session.trackId,
+            trackTitle: event.tracks.find((track) => track.id === session.trackId)?.name,
+            startTime: session.dates?.start?.toISO(),
```

Comment:
@HugoGresse I have `undefined` values for `et-si-on-implementait-une-mach` session's `startTime` and `endTime` fields : is it normal ?

Should I ignore this session ?

Reply:
done